File: common/src/main/java/guepardoapps/lucahome/common/utils/ByteStringHelper.java
package guepardoapps.lucahome.common.utils;

public class ByteStringHelper {
    /**
     * @param charArray the charArray  to convert to a string
     * @return returns a converted string from a char array
     */
    public static String ReadStringFromCharArray(char[] charArray) {
        for (int index = 0; index < charArray.length; index++) {
            charArray[index] /= 69;
        }

        return new String(charArray);
    }
}

File: common/src/main/java/guepardoapps/lucahome/common/utils/ByteStringHelper.kt
package guepardoapps.lucahome.common.utils;

import guepardoapps.lucahome.common.extensions.div

class ByteStringHelper {

    companion object {

        /**
         * @param charArray the charArray  to convert to a string
         * @return returns a converted string from a char array
         */
        fun readStringFromCharArray(charArray: CharArray): String {
            for (index in 0..charArray.size) {
                charArray[index] = charArray[index].div(69)
            }

            return String(charArray)
        }
    }
}
